addons/inventory/XEH_preStart.sqf

```diff
@@ -36,14 +36,11 @@ _grenades_ItemList = _grenades_ItemList arrayIntersect _grenades_ItemList;
 uiNamespace setVariable [QGVAR(Grenades_ItemList), _grenades_ItemList];
 
 // generate list of medical items
-private _medical_ItemList = [];
+private _medical_ItemList = ["FirstAidKit", "Medikit"];
```

Comment:
What about the gm items or other hard coded ones? Probably should walk the config instead.

Reply:
now adds from https://github.com/acemod/ACE3/blob/master/addons/medical_treatment/CfgReplacementItems.hpp

Reply:
But this is a different component. It should be able to list all medical items, even when using vanilla medical, because one deleted the medical components.

Reply:
even if you delete medical, compats will still add the array to configs

Reply:
Ok, then you still need to handle the GM items, or do we have a GM compat?

Reply:
adjusted for changes in #7224
GM items not handled in this PR